ci/builder/Dockerfile

```diff
@@ -165,6 +166,14 @@ RUN mkdir rust \
         ;; \
     esac \
     && rm -rf rust \
+    && mkdir rust \
+    && curl -fsSL https://static.rust-lang.org/dist$RUST_DATE/rust-std-$RUST_VERSION-wasm32-unknown-unknown.tar.gz > rust.tar.gz \
+    && curl -fsSL https://static.rust-lang.org/dist$RUST_DATE/rust-std-$RUST_VERSION-wasm32-unknown-unknown.tar.gz.asc > rust.asc \
+    && gpg --verify rust.asc rust.tar.gz \
+    && tar -xzf rust.tar.gz -C rust --strip-components=1 \
+    && rm -f rust.asc rust.tar.gz \
+    && mv rust/rust-std-wasm32-unknown-unknown/lib/rustlib/wasm32-unknown-unknown /usr/local/lib/rustlib/ \
+    && rm -rf rust \
```

Comment:
Can you make this look more like the cases above? Specifically, it'd be more consistent to ditch the `mkdir rust` and `rm -rf rust` and instead embed this above after L163. Also use `tar -C` to unpack directly into `/usr/local/lib/rustlib` (you may need to adjust --strip-components).